plugin/trino-hive/src/main/java/io/trino/plugin/hive/HivePartitionResult.java

```diff
@@ -40,30 +41,41 @@
     private final TupleDomain<ColumnHandle> enforcedConstraint;
     private final Optional<HiveBucketHandle> bucketHandle;
     private final Optional<HiveBucketFilter> bucketFilter;
+    private final List<String> partitionNames;
+    private final boolean loadPartitions;
```

Comment:
It's not very descriptive name? Can it be improved, e.g: `tooManyPartitions` or `canLoadPartitions` (if we need to use this name)?